forward/spike.lua

```diff
@@ -12,10 +12,11 @@ local function runmain()
    end
 
    godefs.Init()
+   local HEALTH_CHECK_HTTP = 1
```

Comment:
Can we make these constants exported by cgo, so that we don't have to define them in two places?

Reply:
Good idea, I'll get to it when I'm less hosed.

Reply:
Turns out cgo can't export constants, only functions. It's possible to define a new exported function for every field in the enum (or export a function that converts a string into an integer), but it sounds like it'd just make things uglier. For now, I'll just move those constants into a separate Lua file.